packages/react-charting/src/components/AreaChart/AreaChart.base.tsx

```diff
@@ -890,15 +890,15 @@ export class AreaChartBase extends React.Component<IAreaChartProps, IAreaChartSt
     });
   };
 
-  private _getAriaLabel = (lineIndex: number, pointIndex: number): string => {
+  private _getAriaLabel(lineIndex: number, pointIndex: number): string {
```

Comment:
I think this change is not needed.

Reply:
yes, this change is required as we have to mock the getAriaLabel function for Area chart and if we keep the function anonymous using the arrow function, jest cannot spy on that function.